docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -104,15 +96,15 @@ The next version did introduce some interesting new features:
 - [Generic covariant and contravariant](../../standard/generics/covariance-and-contravariance.md)
 - [Embedded interop types](../../framework/interop/type-equivalence-and-embedded-interop-types.md)
 
-Embedded interop types alleviated a deployment pain. Generic covariance and contravariance give you more power to use generics, but they're a bit academic and probably most appreciated by framework and library authors. Named and optional parameters let you eliminate many method overloads and provide convenience. But none of those features are exactly paradigm altering.
+Embedded interop types eased a deployment pain. Generic covariance and contravariance give you more power to use generics, but they're a bit academic and probably most appreciated by framework and library authors. Named and optional parameters let you eliminate many method overloads and provide convenience. But none of those features are exactly paradigm altering.
```

Comment:
I don't see the benefit of the first sentence at all. Can we talk a little bit about the problem? Give them something to search for if they're interested in the topic. What if the generics area just said "Generics solved a common problem with types" and left it at that 😁 

Something like? (i don't know if this is technically correct or not, I just skimmed the framework article)

```suggestion
Embedded interop types eased the deployment pain of creating COM interop assemblies for your application. Generic covariance and contravariance give you more power to use generics, but they're a bit academic and probably most appreciated by framework and library authors. Named and optional parameters let you eliminate many method overloads and provide convenience. But none of those features are exactly paradigm altering.
```